_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -235,6 +242,12 @@ public function getCategoryOptions() {
     return $categories;
   }
 
+  /**
+   * Please update this function description..
```

Comment:
Formats a Wordpress post response from WP JSON API into a format we want in Drupal